src/main/java/org/broadinstitute/hellbender/utils/Nucleotide.java

```diff
@@ -3,22 +3,49 @@
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
-import java.util.stream.LongStream;
 
 /**
  * Represents the nucleotide alphabet with support for IUPAC ambiguity codes.
  *
  * <p>
  *    This enumeration not only contains standard (non-ambiguous) nucleotides, but also
- *    values to represent ambiguous and invalid codes.
+ *    values to represent ambiguous and an invalid nucleotide call ({@link #X} aka {@link #INVALID}.
```

Comment:
Perhaps "...but also contains ambiguous nucleotides, as well as a code {@link #X} (a.k.a. {@link #INVALID}) for invalid nucleotide calls."